src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -87,11 +86,13 @@ public void run(RunConfiguration config) throws AppEngineException {
               .getAbsolutePath();
       jvmArguments.add("-javaagent:" + appengineAgentJar);
     } 
-    arguments.addAll(DevAppServerArgs.get("server", config.getHost()));
-    arguments.addAll(DevAppServerArgs.get("address", config.getPort()));
-    arguments.addAll(DevAppServerArgs.get("jvm_flag", config.getJvmFlags()));
```

Comment:
It definitely doesn't.  The easy way to test is to run:
```
$ java -cp $GOOGLE_CLOUD_SDK_HOME/platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar com.google.appengine.tools.development.DevAppServerMain [flags] /tmp
```
For flags that aren't accepted, the command will show the help text.  For flags that are accepted, the command will run and spit out some error messages of some kind:
```
$ java -cp $GOOGLE_CLOUD_SDK_HOME/platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar com.google.appengine.tools.development.DevAppServerMain /tmp
********************************************************
There is a new version of the SDK available.
-----------
Latest SDK:
Release: 1.9.49
Timestamp: Wed Jan 18 12:33:47 EST 2017
API versions: [1.0]
[…]
```

Running with `-jvm_flag=foo` results in the help text:
```
$ java -cp $GOOGLE_CLOUD_SDK_HOME/platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar com.google.appengine.tools.development.DevAppServerMain -jvm_flag=foo /tmp
Usage: <dev-appserver> [options] <app directory>

Options:
 —help, -h                 Show this help message and exit.
[…]
```